flake-info/src/data/import.rs

```diff
@@ -125,7 +126,9 @@ pub struct Package {
     pub pname: String,
     pub version: String,
     #[serde(default)]
-    pub outputs: HashMap<String, String>,
+    pub outputs: HashMap<String, Option<String>>,
```

Comment:
Why is it `Option<String>` as value type?

Reply:
See https://github.com/NixOS/nix/pull/6242#issuecomment-1066567786, we settled on making the output paths nullable.